### PR TITLE
chore: a11y fixes

### PR DESCRIPTION
**Pull request by davidnuescheler**
https://www.hlx.live/home

vs. 

https://a11y-fixes--helix-website--adobe.hlx.live/home

### CODE REVIEW

**Comment by davidnuescheler** [comment]
vs. 
image

File: blocks/header/header.js [hunk 1]
```diff
@@ -133,11 +133,6 @@ class Gnav {
 
     const brand = brandBlock.querySelector('a');
     brand.classList.add('gnav-brand', 'link-highlight-colorful-effect-hover-wrapper');
-
-    // accessibility
-    const brandAriaLabel = brandBlock.textContent ? brandBlock.textContent.trim() : 'Adobe Franklin';
-    brand.setAttribute('aria-label', brandAriaLabel);
```
Comment (fkakatie) [inline blocks/header/header.js:139]: I think you would only ever need to add an `aria-label` if the element had no text content...?

**Review by fkakatie — APPROVED** [review]

File: blocks/header/header.js (hunk 1 above)
Comment (davidnuescheler) [reply blocks/header/header.js:139]: would it still be needed if there is an `alt-text` or `svg` `title`?

Comment (fkakatie) [reply blocks/header/header.js:139]: Good point. No, I don't think it would be.

**Pull request merged**